frontend/src/components/bedpres-preview.tsx

```diff
@@ -18,28 +18,31 @@ const BedpresPreview = ({ bedpres, registrationCounts }: Props): JSX.Element =>
 
     return (
         <LinkBox data-testid={bedpres.slug}>
-            <Flex
-                alignItems="center"
-                border="2px"
-                borderColor="transparent"
-                p={[0, null, null, null, 5]}
-                _hover={{ bg: hoverColor }}
-            >
-                <Box position="relative">
-                    {/** Parent box is required to prevent child box from scaling inconsistently */}
-                    <Box pos="relative" overflow="hidden" borderRadius="50%" w="85px" h="85px">
-                        <Image src={logoUrl} alt={bedpres.title} layout="fill" objectFit="fill" />
+            <Flex alignItems="center" p={[0, null, null, null, 5]} _hover={{ bg: hoverColor }}>
+                <Flex alignItems="center" gap="5">
+                    <Box display={['none', 'block']}>
```

Comment:
er mer fan av logo i stedet for tekst på mobil egentlig, vet ikke hva dere tenker @boaanes @alvarhonsi 
![2023-01-15_15:14:02](https://user-images.githubusercontent.com/6720179/212545910-1d059229-6ed0-4910-a8d4-f70cd21167b0.png)

Reply:
Jeg er litt splittet der, men det var mer med tanke på a11y.

Reply:
@boaanes hva tenker du?

Reply:
Jeg liker logo, hvis det ikke er en beskrivende nok logo er det bare å gå inn på eventet

Reply:
Vi kan ta det på møte, så kan jeg bare reverte om ingen liker. Tar ikke lang tid.